### ISSUE TITLE
Pre-trained models availability

### ISSUE THREAD

**Issue by Cameltr** [Stillrot/RGINP#1]
Hello, thank you for your excellent work！ When will the pre-trained models be available?

**Comment by gibiee** [comment]
There is no pretrained weight file in the repo. So, I trained this model myself. I share it, so please use it.

https://drive.google.com/drive/folders/1ejrdlaXiriUrM67Oy0a9ccUx8nQYDt6a?usp=sharing

Check the following when running the test code.
1. If a data type `np.int` error occurs, you should modify line 123 in rginp.py. `np.int` to `np.int32`
2. In README, test images are `sample.jpg`, but it actually works only when they are `28000.jpg` like CelebA_HQ dataset.

**Comment by Mr-Nobody-dey** [comment]
Hi gibiee,
How well your model performs? 
and what does point 2. "In README, test images are sample.jpg, but it actually works only when they are 28000.jpg like CelebA_HQ dataset." means? does it mean that I should have 28000 images as test images, only then it will work?

**Comment by gibiee** [comment]
@Mr-Nobody-dey 
When I tested, the results seem natural. But, I don't know if it reflected the reference image sufficiently.
And point 2 means just file naming. It should be named by number because code will load the label data by the number.
